### PR TITLE
New tech note: drake version 7.0.0

### PR DESCRIPTION
**Pull request by wlandau**
I plan to submit [`drake`](https://github.com/ropensci/drake) version 7.0.0 to CRAN on March 10 (ref: https://github.com/ropensci/drake/issues/753). There are exciting new features and performance improvements, and I would like to share them in the enclosed tech note. If this sounds good, would March 18 be a good publication date? The CRAN binaries should be available by then.

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Sounds good in principle @wlandau. Karthik will review the tech note before your proposed March 18th publication date.

**Comment by wlandau** [comment]
Exciting update: `drake` version 7.0.0 was accepted to CRAN yesterday, and the builds are in progress. This tech note draft is up to date with the release.

**Comment by jeroen** [comment]
The "disclaimer" section is a bit silly because ropensci is not your employer and and your employer is also not mentioned otherwise in the text. How could anyone possible think this post represents the official views of my employer? I think you should remove this.

**Comment by wlandau** [comment]
I agree that risk is low. However, my employer [does technically own `drake`](https://github.com/ropensci/drake/blob/ae093a068c1be05bcb217de07b79882c0643e156/DESCRIPTION#L70-L73), and they take even small risks very seriously. An explicit disclaimer is important from where I am standing.

File: content/technotes/2019-03-18-drake-700.Rmd [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+```{r install, eval = FALSE}
+install.packages("drake")
+```
+
+## Recap
+
+Data analysis can be slow. A round of scientific computation can take several minutes, hours, or even days to complete. After it finishes, if you update your code or data, your hard-earned results may no longer be valid. How much of that valuable output can you keep, and how much do you need to update? How much runtime must you endure all over again?
+
+For projects in R, the [`drake`](https://github.com/ropensci/drake) package can help. It [analyzes your workflow](https://ropenscilabs.github.io/drake-manual/plans.html), skips steps with up-to-date results, and orchestrates the rest with [optional distributed computing](https://ropenscilabs.github.io/drake-manual/hpc.html). At the end, [`drake`](https://github.com/ropensci/drake) provides evidence that your results match the underlying code and data, which increases your ability to trust your research.
```
Comment (karthik) [inline content/technotes/2019-03-18-drake-700.Rmd]: _(minor word smithing)_ For projects in R, the workflow package `drake` can help.

File: content/technotes/2019-03-18-drake-700.Rmd [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+|----------------|-----------------------------|
+| `map()`        | `pmap()` from `purrr`       |
+| `cross()`      | `crossing()` from `tidyr`   |
+| `combine()`    | `summarize()` from `dplyr`  |
+
+These transformations help declare batches of multiple targets.
+
+```{r transformations}
+library(drake)
+source <- c("census", "gapminder")
```
Comment (karthik) [inline content/technotes/2019-03-18-drake-700.Rmd]: What type of data sources are these? Have they already been read into objects? Just a short R comment will be helpful to guide the reader.

**Comment by karthik** [comment]
Hi Will! This post reads really well to me and I don't have much in the way of edits or request for changes. Other than the minor clarifications noted above, I'm happy with this post!

**Comment by wlandau** [comment]
Thanks, @karthik! I believe I have addressed your comments in f01aa34541689433b2e4daade79de0c0b0343207 and 878fe8ee11b407cf2076a17c5339e07f0431b618. Please let me know if you think of anything else.

**Comment by wlandau** [comment]
By the way, all the checks and binaries completed on CRAN without a hitch. The launch of `drake` 7.0.0 itself was successful.

**Comment by stefaniebutland** [comment]
@wlandau fyi, Planning to publish this and tweet on Monday March 18th AM.

**Comment by wlandau** [comment]
Fantastic, @stefaniebutland! Thank you.

**Comment by wlandau** [comment]
Thanks again @karthik and @stefaniebutland! It is exciting to see this post in production.

**Pull request merged**